vhdbuilder/packer/init-variables.sh

```diff
@@ -82,6 +82,14 @@ if [ -z "${SUBNET_NAME}" ]; then
 	SUBNET_NAME="packer"
 fi
 
+SHALLOW_REPLICATION_TOGGLE="false"
+if [[ "${MODE}" == "linuxVhdMode" ]] && [[ "${ENVIRONMENT,,}" == "test" ]]; then
+  # continue using full replication for prod builds for now
+  SHALLOW_REPLICATION_TOGGLE="true"
```

Comment:
after reading this once again: https://learn.microsoft.com/en-us/azure/virtual-machines/shared-image-galleries?tabs=vmsource%2Cazure-cli#shallow-replication, I don't personally think a toggle is actually necessary, should be safe to just enable to all our environments